packages/web/src/services/sentry.ts

```diff
@@ -43,7 +45,7 @@ export const initializeSentry = () => {
       }
       // filter out analytics events
       if (hint && hint.xhr) {
-        const url = hint.xhr.__sentry_xhr__.url
+        const url = hint.xhr.__sentry_xhr_v3__.url
```

Comment:
ah interesting

Reply:
Yeah turns out this isn't a documented parameter and they can change it whenever they like! 😬